pallets/social/src/lib.rs

```diff
@@ -24,7 +21,7 @@ pub struct Blog<T: Trait> {
 
   // Can be updated by the owner:
   pub writers: Vec<T::AccountId>,
-  pub slug: Vec<u8>,
+  pub handle: Vec<u8>,
```

Comment:
Hande should be optional.
```suggestion
  pub handle: Option<Vec<u8>>,
```